TypeScript/buttonConfirmation.ts

```diff
@@ -87,7 +94,7 @@ export function confirmation(
 
 				collector.on("ignore", async (i) => {
 					await i.reply({
-						content: options?.wrongUserResponse ?? "Not for you!",
+						content: options?.wrongUserResponse,
```

Comment:
could remove the ? in ?. but its fine

Reply:
![image](https://user-images.githubusercontent.com/85353424/192245594-88dce4e7-6872-4800-800f-1ca37c257168.png)